lib/tasks/cortex/core/media.rake

```diff
@@ -29,15 +29,17 @@ namespace :cortex do
                            },
                          metadata:
                            {
-                             styles: {
-                               large: {geometry: '1800x1800>', format: :jpg},
-                               medium: {geometry: '800x800>', format: :jpg},
-                               default: {geometry: '300x300>', format: :jpg},
-                               mini: {geometry: '100x100>', format: :jpg},
-                               micro: {geometry: '50x50>', format: :jpg},
-                               post_tile: {geometry: '1140x', format: :jpg}
+                            image_styles: {
```

Comment:
This breaks from my original intention in designing the syntax for asset config - I wanted it to mimic Paperclip's config almost 1:1. What's wrong with using the `styles` key? In the `format_data` method, just delete the `styles` and 'processors' hash keys if you need to remove styles for a non-post-processed asset.